### DISCUSSION TITLE
Will it be javascript package?

### DISCUSSION THREAD

**Discussion by romdevios** [apple/pkl#31] [Q&A]
I need the same mechanic for my javascript project. Is it worth to wait? At least, some instrument to easy conversion to readable for javascript format.

**Comment by jasongwartz** [comment]
Resposting from: https://github.com/apple/pkl/issues/15#issuecomment-1925742364

> If anyone would like to contribute to discussions about Pkl support for Node and TypeScript, please join [the unofficial community Discord](https://pkl-playground.vercel.app/discord-invite) and join the conversation in #bindings-typescript!

**Reply by xav-ie** [reply]
link expired :(

**Reply by jasongwartz** [reply]
Sorry about that, @xav-ie - I've set up a redirect that we can maintain (in case of expiry), please now use: https://pkl-playground.vercel.app/discord-invite

**Reply by jasongwartz** [reply]
I've also updated the referenced comment for future people who end up here

**Reply by xav-ie** [reply]
Thank you :)
